core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java

```diff
@@ -114,10 +112,12 @@ public void calculateNonaxialForces(FlightConditions conditions, Transformation
 		// Multiple fins with fin-fin interference
 		double cna;
 		double theta = conditions.getTheta();
-		double angle = baseRotation + transform.getXrotation();
```

Comment:
Does transform.getXrotation completely subsume baseRotation, ie is that member not needed any more?

Reply:
yea, I think so.   But I want further testing to verify that it behaves the appropriate way everywhere else, too.